src/templates/admin/core/manager/users/enrol_users.html

```diff
@@ -0,0 +1,94 @@
+{% extends "admin/core/base.html" %}
+{% load static from staticfiles %}
+
+{% block breadcrumbs %}
+    {{ block.super }}
+    <li><a href="{% url 'core_manager_index' %}">Manager</a></li>
+    <li><a href="{% url 'core_manager_users' %}">Enrol Users</a></li>
+{% endblock %}
+
+{% block title %}Enrol Users{% endblock %}
+{% block title-section %}Enrol Users{% endblock %}
+{% block title-sub %}Search for an enrol user accounts in your journal.{% endblock %}
+
+{% block body %}
+    <div class="large-12 columns">
+        <div class="box">
+            <div class="title-area">
+                <h2>Enrol Users</h2>
+                <a href="{% url 'core_manager_index' %}" class="button">< Go back</a>
```

Comment:
I think we link to this page from various places now, we might need a GET parameter so we can return them to the right place.